pkg/kubelet/hosts_test_file1

```diff
@@ -0,0 +1,8 @@
+# hosts file for testing.
```

Comment:
Can we not add test files in the pkg/kubelet? The test itself should write a temporary file and clean up after itself.